### PR TITLE
[modules] Add new covid tracker module

### PR DESCRIPTION
**Pull request by dabcoder**
This is a covid tracker module using this API: https://coronavirus-tracker-api.herokuapp.com/#/v2/. 

the config would look like:

```yaml
covid:
  enabled: true
  countries:
    - "FR"
    - "US"
    - "DE"
    - ...
   refreshInterval: 120
   position:
        ...
```

And the result would look like:

```
┌─────────────────────── Covid tracker ────────────────────────┐
│  ││Gobal
│  ││Confirmed: ...
│  ││Deaths: ...
│  ││Country: FR
│  ││Confirmed: ...
│  ││Deaths: ...
│  ││Country: US
│  ││Confirmed: ...
│  ││Deaths: ...
```

Remaining issue: the third country stats do not appear here, I tried setting `widget.View.SetScrollable` to true but that didn't seem to change anything.

### CODE REVIEW

**Comment by senorprogrammer** [comment]
Could you describe the issue with the third country in more detail?  I'm running this locally and its looking good. To make the widget scrollable to see the rest of my data, I used this config:

```yaml
    covid:
      countries:
        - "FR"
        - "CA"
        - "US"
        - "DE"
      enabled: true
      focusable: true
      position:
        top: 3
        left: 2
        height: 1
        width: 1
      refreshInterval: 120
```

File: modules/covid/widget.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		title = widget.CommonSettings().Title
+	}
+
+	cases, err := LatestCases()
+	var covidStats string
+	if err != nil {
+		widget.err = err
+	} else {
+		// Display global stats
+		covidStats = fmt.Sprintf("[%s]Gobal[white]\n", widget.settings.Colors.Subheading)
```
Comment (senorprogrammer) [inline modules/covid/widget.go:57]: sp `Global`

File: modules/covid/widget.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		covidStats += fmt.Sprintf("%s: %d\n", "Confirmed", cases.Latest.Confirmed)
```
Comment (senorprogrammer) [inline modules/covid/widget.go:58]: I suspect that Confirmed and Deaths in this case should not be the same value.

File: modules/covid/widget.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	// Retrieve country stats if country codes are set in the config
+	if len(widget.settings.countries) > 0 {
+		countryCases, err := widget.LatestCountryCases(widget.settings.countries)
+		if err != nil {
+			widget.err = err
+		} else {
+			for i, name := range countryCases {
+				covidStats += fmt.Sprintf("[%s]Country[white]: %s\n", widget.settings.Colors.Subheading, widget.settings.countries[i])
+				covidStats += fmt.Sprintf("%s: %d\n", "Confirmed", name.Latest.Confirmed)
+				covidStats += fmt.Sprintf("%s: %d\n", "Deaths", name.Latest.Deaths)
```
Comment (senorprogrammer) [inline modules/covid/widget.go:70]: For formatting numbers for easy readability, take a look at https://github.com/wtfutil/wtf/blob/master/modules/github/display.go#L126

It'll format the number according to the user's locale (in my case, like `554,282`). 

Fix these minor issues and I'd be happy to merge this PR in.

**Review by senorprogrammer — CHANGES_REQUESTED** [review]

File: modules/covid/widget.go (hunk 1 above)
Comment (dabcoder) [reply modules/covid/widget.go]: Addressed in 46de9e53.

File: modules/covid/widget.go (hunk 2 above)
Comment (dabcoder) [reply modules/covid/widget.go:58]: Also addressed in 46de9e53.

File: modules/covid/widget.go (hunk 3 above)
Comment (dabcoder) [reply modules/covid/widget.go]: Gave it a try in 5934ed21, but I also hardcoded it to `language.English` like in the GitHub module. Is there a way to make this configurable, from the config yaml file maybe?

**Comment by senorprogrammer** [comment]
Good point, I've removed the hard-coded English localization and made it configurable via a top-level `language` configuration key in this PR: https://github.com/wtfutil/wtf/pull/1038

Take a look here for how to implement the solution now: https://github.com/wtfutil/wtf/pull/1038/files#diff-1d37a88633fac3cd7f161204812dbf98d0267835aef4d6d5bc740a1bfe4163f7R125

**Comment by senorprogrammer** [comment]
Merged your branch and updated the localization here https://github.com/wtfutil/wtf/pull/1037. You should be able to set yours in your config file now.

**Comment by dabcoder** [comment]
> Merged your branch and updated the localization here #1037. You should be able to set yours in your config file now.

Thanks, much appreciated, will have a look and aim to do that in a subsequent PR.

> Could you describe the issue with the third country in more detail? I'm running this locally and its looking good. To make the widget scrollable to see the rest of my data, I used this config...

I used this config and didn't manage to display the third country stats:

```yaml
covid:
      enabled: true
      focusable: true
      countries:
        - "FR"
        - "US"
        - "DE"
      position:
        top: 1
        left: 2
        height: 1
        width: 2
      refreshInterval: 120
```

**Comment by reportaman** [comment]
@dabcoder Thanks! Is it possible to specify state/province/region/city/town in the location for covid tracker?

**Comment by dabcoder** [comment]
@reportaman sorry I missed your question. Just the country codes for now, this is the only option at the moment. You're right, I need to issue a PR for the documentation part.

**Pull request merged**